functions/quality_assessment/quality_assessment.py

```diff
@@ -145,12 +152,41 @@ def generate_message(bedrock_runtime: boto3.client,
     return response_body
 
 
-def get_prompt_list() -> list[dict]:
+@app.post("/update-prompt")
+@tracer.capture_method
+def update_prompt() -> dict:
```

Comment:
If you are feeling fancy, you could add some validation of the incoming request body. 
[Validating payloads](https://docs.powertools.aws.dev/lambda/python/latest/core/event_handler/api_gateway/#validating-payloads)